runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -596,15 +623,22 @@ void sendNotification(RuneScapeProfile profile, PatchPrediction prediction, Farm
 			}
 		}
 
+		AppendCropInfo(stringBuilder, prediction.getCropState(), prediction.getProduce(), patch);
+
+		notifier.notify(stringBuilder.toString());
+	}
+
+	void AppendCropInfo(StringBuilder stringBuilder, CropState cropState, Produce produce, FarmingPatch patch)
```

Comment:
why was this changed?

Reply:
GeomancyManager also needs to send a notification (about the disease state), so rather than have the logic copy pasted in two places I broke it out into a function.